mobile_verifier/tests/integrations/common/mod.rs

```diff
@@ -179,35 +180,27 @@ pub fn seconds(s: u64) -> std::time::Duration {
     std::time::Duration::from_secs(s)
 }
 
+pub fn mock_hex_boost_data_default() -> HexBoostData<Assignment, Assignment, Assignment> {
+    HexBoostData::builder()
+        .urbanization(Assignment::A)
+        .footfall(Assignment::A)
+        .landtype(Assignment::A)
+        .build()
+        .unwrap()
+}
+
 type MockAssignmentMap = HashMap<hextree::Cell, Assignment>;
 
-#[derive(Default)]
-pub struct MockHexAssignments {
+#[allow(dead_code)]
```

Comment:
Since https://github.com/helium/oracles/pull/804 landed, you can remove `#[allow(dead_code)]` from this file. 
This function is unused. 
`MockHexBoostingClient` is used.